Gemfile

```diff
@@ -10,11 +10,14 @@ gem "sprockets-rails"
 gem "turbo-rails"
 gem "importmap-rails"
 gem "redis"
-gem "capybara"
-gem "selenium-webdriver"
-gem "web-console", group: :development
 
-# Start debugger with binding.b [https://github.com/ruby/debug]
-# gem "debug", ">= 1.0.0"
+group :development, :test do
+  gem "capybara"
+  gem "selenium-webdriver"
+  gem "standardrb", "~> 1.0"
+  gem "debug", ">= 1.0.0"
+end
```

Comment:
just a quick question, why do we prefer adding development dependencies here instead of using the signalman.gemspec?

Reply:
I think I know that one: this is not really necessary because (AFAIK), gems listed in the Gemfile are not actually required by the library, only gems under `.gemspec` file. whatever is listed here will always be only for development/test

Reply:
Yeah, it's more flexible in the Gemfile, like if I want to deploy Signalman's test/dummy app as a demo for people to try out. The dev_dependencies don't really do much other than list it in Rubygems.org and that's not that useful.

Reply:
cool! feel free to close it if doesn't add value, not a problem :rocket: